src/main/resources/csar/tosca.yml

```diff
@@ -1135,6 +1135,43 @@ data_types:
           More info: http://kubernetes.io/docs/user-guide/volumes#emptydir
         required: false
 
+  org.alien4cloud.kubernetes.api.datatypes.volumes.CephFSVolumeSpec:
+    derived_from: tosca.datatypes.Root
+    properties:
+      monitors:
+        type: list
+        entry_schema:
+          type: string
+        description: |
+          Required: Monitors is a collection of Ceph monitors. More info: https://github.com/kubernetes/examples/blob/master/volumes/cephfs/README.md
```

Comment:
May not be needed to have the description start with "Required: " or "Optional:" as it is seen in the UI which is taking into account the value of the property "required" (and other data types descriptions don't use this convention)

Reply:
You're right! Updated.